ssh/tunnel.go

```diff
@@ -177,6 +179,19 @@ func (t *Tunnel) Stop() error {
 	return nil
 }
 
+func (t *Tunnel) startKeepalive(user string, sshConnection *ssh.Client) {
+	ticker := time.NewTicker(2 * time.Second)
+	defer ticker.Stop()
+	for {
+		<-ticker.C
+		keepaliveName := "keepalive@github.com/alphagov/paas-cf-conduit"
```

Comment:
This feels like something that should be extracted into a constant.

Reply:
It's now in a constant at the top of the file